src/flags.ts

```diff
@@ -37,7 +37,7 @@ export const help = (opts: Partial<BooleanFlag<boolean>> = {}) => {
     description: 'Show CLI help.',
     ...opts,
     parse: async (_: any, cmd: Command) => {
-      new Help(cmd.config).showHelp(cmd.argv)
+      new Help(cmd.config).showHelp([cmd.id!, ...cmd.argv])
```

Comment:
I'm not sure if it matters or not, but I have seen scenarios where a command doesn't have an ID, particularly when we do weird things with them in UT.  

https://github.com/salesforcecli/plugin-source/blob/f5f681b19bcae6a98c25e45e7b461a624f1c2920/test/commands/source/cancel.test.ts#L43

One of those "I'm super cautious about asserting things to TS"